Oauth (#53)
deity-io/falcon-docs: docs/backend/installing-magento2.md

      "config": {
        "host": "your-magento-host-url-here",
        "protocol": "https",
        "auth": {
          "type": "admin-token",
          "username": "your-admin-username",
          "password": "your-admin-password"
        }
      }
    }
  }
}
```

##### Connect Falcon to Magento through oAuth

> To connect through oAuth you need to create an new Magento integration:

Reviewer: ```suggestion
To connect through oAuth you need to create a new Magento integration:
```

* sign in to Magento Admin panel
* go to *System* / *Extensions* / *Integrations* and press *Add new Integration*)

> * create new *Integration* via providing its *Name* and specifying *Resource Access: All*

Reviewer 2: @tutok is it safe to provide `ALL` access? (@sergy-gzh?)

Author: @Borales what do you think about this:
```suggestion
* create new *Integration* via providing its *Name* and specifying *Resource Access* of your choice.
```

> * create new *Integration* via providing its *Name* and specifying *Resource Access: All*

Reviewer 3: ```suggestion
* Fill in the *Name* and go to  *Role Resources* tab, set *Resource Access* to *Custom* and check the following permissions in the list below:
** Sales
** Catalog
** Stores -> Settings -> All stores
** Global Search
```

> * *Activate* an integration, grab all *Integration Tokens* and update Falcon `api-magento2.config.auth` section:

Reviewer 2: ```suggestion
* *Activate* an integration, grab all *Integration Tokens* and update Falcon-Server `apis.magento2.config.auth` config section:
```

      "config": {
        "host": "your-magento-host-url-here",
        "protocol": "https",
        "auth": {
          "type": "admin-token",
          "username": "your-admin-username",
          "password": "your-admin-password"
        }
      }
    }
  }
}
```

##### Connect Falcon to Magento through oAuth

To connect through oAuth you need to create a new Magento integration:
* sign in to Magento Admin panel
* go to *System* / *Extensions* / *Integrations* and press *Add new Integration*)
* Fill in the *Name* and go to  *Role Resources* tab, set *Resource Access* to *Custom* and check the following permissions in the list below:

> ** Sales

Reviewer 3: ```suggestion
    * Sales
```

> ** Catalog

Reviewer 3: ```suggestion
    * Catalog
```

> ** Stores -> Settings -> All stores

Reviewer 3: ```suggestion
    * Stores -> Settings -> All stores
```

> ** Global Search

Reviewer 3: ```suggestion
    * Global Search
```
